lib/Runtime/Base/FunctionBody.h

```diff
@@ -1179,19 +1179,19 @@ namespace Js
     struct LoopHeader
```

Comment:
Double checked and this seems ok for now. "FORCE_USE_WRITE_BARRIER" starts after this class definition at line 1298. (I avoided splitting this file to prevent merge hell.)

---
In reply to: [89894881](https://github.com/Microsoft/ChakraCore/pull/2087#discussion_r89894881) [](ancestors = 89894881)